bugbug/models/duplicate.py

```diff
@@ -120,7 +124,7 @@ def get_labels(self):
         # Store all remaining ids
         non_duplicate_ids = list(all_ids - set(duplicate_ids))
 
-        print(f"Number of duplicate labels is: {self.num_duplicates}")
+        logger.info("Number of duplicate labels is: %s", self.num_duplicates)
```

Comment:
```suggestion
        logger.info("Number of duplicate labels is: %d", self.num_duplicates)
```


When the value is an integer, please use `%d`. Please apply the same change where applicable.

Reply:
thanks for pointing this mistake out, i will do the changes